controllers/front/hostedIframe.php

```diff
@@ -103,4 +105,25 @@ public function setMedia()
             );
         }
     }
+
+    protected function getDefaultTermsAndConditions()
```

Comment:
is it even used? what happens if you just add false or null? Does it even have any impact?

Reply:
It is, it provides Terms and Conditions 
![image](https://user-images.githubusercontent.com/97019420/186133434-2606c216-d6ba-4eb9-92f7-8b399b94b3da.png)
If I use false or null form do not crash it's just there not be any terms and conditions 
![image](https://user-images.githubusercontent.com/97019420/186133647-152f87c6-1f1c-4fb0-b960-8d8076740496.png)

So or I just leave copied functionality of tos_cms or insert false and leave that place blank? @margud @tomjas1997

Reply:
i think its ok